packages/cozy-mespapiers-lib/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "cozy-mespapiers-lib",
+  "version": "0.1.0",
+  "description": "Cozy Mes papiers lib",
+  "license": "AGPL-3.0",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "babel src -d dist --copy-files --verbose",
+    "start": "yarn build --watch",
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@testing-library/react": "11.2.7",
+    "babel-preset-cozy-app": "2.0.1",
+    "cozy-bar": "7.16.0",
+    "eslint-config-cozy-app": "^4.0.0",
+    "git-directory-deploy": "1.5.1",
+    "react-test-renderer": "16.14.0",
+    "redux-mock-store": "1.5.4",
+    "cozy-client": "^27.22.0",
+    "react-router-dom": "5.2.0",
+    "cozy-ui": "^62.0.0",
+    "cozy-intent": "^1.13.5",
+    "cozy-realtime": "^4.0.2",
+    "jest-environment-jsdom-sixteen": "2.0.0",
+    "jest-resolve-cached": "1.1.1"
+  },
+  "dependencies": {
+    "@date-io/date-fns": "1",
+    "@material-ui/core": "4",
+    "@material-ui/pickers": "3.3.10",
+    "cozy-device-helper": "^1.17.0",
+    "cozy-doctypes": "^1.83.3",
+    "cozy-harvest-lib": "^7.1.1",
+    "cozy-sharing": "^4.0.2",
```

Comment:
According to @Crash-- ,  the library of Cozy should not add any other library. Those librairies should only be devDep + peerDep.


I will add this inside the cozy-guidelines.

Reply:
J'ai tenté une nouvelle approche, ça fonctionne pas dans l'app, je fais une issue